python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -151,3 +139,53 @@ def __call__(self, df, **kwargs):
         sourceType = np.zeros(len(df))
         sourceType[gals] = 2
         return sourceType
+
+
+class UnknownIdentifier(DataFrameAction):
+    """Identifies un classified objects from the dataFrame and marks them as a
+       9 in the added sourceType column"""
+
+    band = Field(doc="The band the object is to be classified as an unkown in.",
+                 default="i",
+                 dtype=str)
+
+    @property
+    def columns(self):
+        return [self.band + "Extendedness"]
+
+    def __call__(self, df, **kwargs):
+        """Identifies sources classed as unknowns"""
+        unknowns = (df[self.band + "Extendedness"] == 9.0)
+        sourceType = np.zeros(len(df))
+        sourceType[unknowns] = 9
+        return sourceType
+
+
+class CoaddPlotFlagSelector(DataFrameAction):
+    """The flags to use for selecting sources for coadd QA"""
```

Comment:
Please be a lot more explicit about what QA means here. Convince me, the clueless user, these are the flags I probably want to plot. What if I want to configure this and use different ones instead?